Contribute/validation-reference/ms-date-invalid.md

```diff
@@ -0,0 +1,25 @@
+---
+title: ms-date-invalid
+description: Explanation and resolution for Docs build issue ms-date-invalid
+author: meganbradley
+ms.author: mbradley # Microsoft employees only
+ms.topic: error-reference
+ms.date: 1/15/2019
+ms.prod: non-product-specific
+---
+# ms-date-invalid
+
+**Coming soon!**
+
+[!INCLUDE [suggestion-note](includes/suggestion-note.md)]
+
+## Suggestion
+
+`Invalid value for ms.date: '{value}'. Must be a date in format MM/DD/YYYY.`
```

Comment:
Does this apply to localized content? This date format is only valid in the US. Think using ISO dates has been raised before

Reply:
It has, but no change to the requirement has been made so far, and we still require MM/DD/YYYY. I don't know whether this will change in the future.